apps/discord-bot/src/commands/vampirez/vampirez.profile.tsx

```diff
@@ -53,9 +56,11 @@ export const VampireZProfile = ({
         time={time}
       />
       <Table.table>
-        <VampireZRow mode="§6Overall" stats={vampirez.overall} t={t} />
-        <VampireZRow mode="§eHuman" stats={vampirez.human} t={t} />
-        <VampireZRow mode="§4Vampire" stats={vampirez.vampire} t={t} />
+        <Table.tr>
+          <VampireZRow mode="§6Overall" stats={vampirez.overall} t={t} />
```

Comment:
Rename to Column because thats what it is.

```suggestion
          <VampireZColumn mode="§6Overall" stats={vampirez.overall} t={t} />
```